### PR TITLE
Add linter for unused & undefined styles, fixing them

### PR DESCRIPTION
**Pull request by derekstavis**
This adds a linter for catching unused and undefined classes in CSS files.
Also fixes all errors found in all components and stories.

### CODE REVIEW

File: webapp/src/components/Toolbar/DateInput/classNames.style.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  position: relative;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  padding-right: var(--date-input-inner-spacing);
+  padding-left: var(--date-input-inner-spacing);
+}
+
+.initialPlaceholder {
+  padding: 0 var(--date-input-inner-spacing)  0 0;
```
Comment (felquis) [inline webapp/src/components/Toolbar/DateInput/classNames.style.css]: Unnecessary multiple spaces `spacing)  0 0`

File: webapp/stories/Progress/index.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ const percentages = [0, 25, 50, 75, 100]
 
 storiesOf('Progress', module)
   .add('Linear', () => (
-    <div className={style.showcase}>
+    <div className={style.container}>
```
Comment (felquis) [inline webapp/stories/Progress/index.js]: Looks like it causes a visual regression
visual-regression-linear-progress

Comment (darakeon) [reply webapp/stories/Progress/index.js]: @derekstavis talked about that, he said that it was intended

Comment (derekstavis) [reply webapp/stories/Progress/index.js]: This is in fact intended. There was a mispell in class name, hence the disabled progress was appearing below the normal state.

**Pull request merged**